src/motion/motion.py

```diff
@@ -109,7 +105,8 @@ def consume(self):
                 log.warn("No responder for event %s registered, skipping", event_name)
                 continue
 
-            self.responder_queue.put((event_name, payload))
+            for index in xrange(len(self.responders[event_name])):
+                self.responder_queue.put((event_name, payload, index))
```

Comment:
This is great.  I like the use of the index. 

Small nitpick: let's change tHe order so it's `event_name, index, payload`